src/common/utils.c

```diff
@@ -1681,3 +1689,78 @@ struct timespec timespec_abs_diff(struct timespec t1, struct timespec t2)
 	res.tv_nsec = diff % (uint64_t) NSEC_PER_SEC;
 	return res;
 }
+
+static
+size_t read_proc_meminfo_field(const char *field)
+{
+	size_t ret, value_kb, value;
```

Comment:
`value_kb` can be moved to the loop to reduce its scope.

Reply:
done.